.github/workflows/node.yml

```diff
@@ -37,3 +37,12 @@ jobs:
           tag_name: ${{ github.event.inputs.network }}-${{ github.event.inputs.tag }}
           target_commitish: ${{ github.event.inputs.network }}
           token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Deploy tracing nodes
```

Comment:
I think you can move this to a single job.

Reply:
I don't think so, I think just update tracing node is enough, general node i think manual update is better, because these have many nodes and distribute many servers. 
but move to single job is better.

Reply:
I mean at least the pangolin testnet.